### PR TITLE
Check in schema.rb and configure Travis to run tests.

### PR DESCRIPTION
**Pull request by jfhamlin**
This gets Travis builds succeeding. Not too hard, since we don't have any tests yet. A few things to note:

1. We haven't yet installed rspec yet, so the test command will change when we move to it.
2. We should eventually ditch .rubocop_todo.yml, excluding rails/bundle files in .rubocop.yml and fixing any violations in our app code.

### CODE REVIEW

**Comment by jfhamlin** [comment]
cc @lelandg559 @jjfreund

File: .rubocop.yml [hunk 1]
```diff
@@ -1,8 +1 @@
-AllCops:
```
Comment (jfhamlin) [inline .rubocop.yml]: We should probably keep this stuff.

Comment (jfhamlin) [inline .rubocop.yml]: This key is deprecated in the latest versions of rubocop.

**Comment by lelandg559** [comment]
LGTM

**Pull request merged**